docs/dev/run-cvo-locally.md

```diff
@@ -0,0 +1,51 @@
+# How to Build and Run CVO Locally
+
+This document provides the instructions to build and run the CVO executable locally. Local execution of CVO, for example executing CVO on your development laptop, allows a "quick-turn" test capability whereby the CVO executable can be quickly rebuilt with changes and launched again without the need of starting a new OCP cluster.
+
+## Preconditions
+
+Build the CVO executable using the standard procedure:
+
+```console
+$ cd ~/go/src/github.com/openshift/cluster-version-operator
+$ make
+```
+
+Start an OCP cluster of the appropriate version. One that is compatiable with the version of CVO with which you're working.
+
+Extract and download the OCP cluster release:
+
+```console
+$ oc image extract -a ~/pull-secret-internal quay.io/openshift-release-dev/ocp-release:4.4.0-rc.4-x86_64 --path /:/tmp/release/
+```
+
+## Steps to Run CVO Locally
+
+Scale down the CVO pod on the cluster:
+
+```console
+$ oc scale --replicas=0 deployment.apps/cluster-version-operator
```

Comment:
We should add the namespace here i.e. ` -n openshift-cluster-version` to the command.

Reply:
Fixed in [3bdbbc0](https://github.com/openshift/cluster-version-operator/pull/381/commits/3bdbbc07d01ff7c4374c039cb68a84564bea7abd)